src/aiida_shell/launch.py

```diff
@@ -32,6 +32,7 @@ def launch_shell_job(  # noqa: PLR0913
     metadata: dict[str, t.Any] | None = None,
     submit: bool = False,
     resolve_command: bool = True,
+    monitors: dict[str, Data] | None = None,
```

Comment:
This should be `Dict`, no?
```suggestion
    monitors: dict[str, Dict] | None = None,
```